internal/worker/task_handler.go

```diff
@@ -86,27 +90,13 @@ func (h *taskHandler) Plan(t sw.StateSwitch, args sw.TransitionArgs) error {
 	}
 }
 
-func (h *taskHandler) ValidatePlan(t sw.StateSwitch, args sw.TransitionArgs) (bool, error) {
-	tctx, ok := args.(*sm.HandlerContext)
-	if !ok {
-		return false, sm.ErrInvalidtaskHandlerContext
-	}
-
-	task, ok := t.(*model.Task)
-	if !ok {
-		return false, errors.Wrap(ErrSaveTask, ErrTaskTypeAssertion.Error())
-	}
-
-	// validate task has action plans listed
-	if len(task.ActionsPlanned) == 0 {
-		return false, errors.Wrap(errTaskPlanValidate, "no task actions planned")
-	}
-
-	// validate task context has action statemachines for execution
-	if len(tctx.ActionStateMachines) == 0 {
-		return false, errors.Wrap(errTaskPlanValidate, "task action plan empty")
-	}
+func (h *taskHandler) ValidatePlan(_ sw.StateSwitch, args sw.TransitionArgs) (bool, error) {
```

Comment:
Given this is a noop at this point, should this method be removed completely?

Reply:
Done.